Setup/UpgradeSchema.php

```diff
@@ -220,10 +220,53 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             }
         }
 
-        $richSnippetsTable = $installer->getConnection()->describeTable($installer->getTable('yotpo_rich_snippets'));
+        if (!$defaultConnection->isTableExists($yotpoRichSnippetsTable)) {
```

Comment:
we already have the code of creating this table - we need to make sure we don't duplicate this code.

Reply:
Hi sharp80. Did you get a chance to read through the pull request description and linked issue? I don't think I understand your comment here. There is code to create the table upon module installation, but not on upgrade. Are you asking for a code change?

Reply:
@fredden - i meant that we have the same code that creates richSnippets table in a different file (InstallScheme.php) and better practice is to move the actual code that creates the table to utility function and call it from the upgradeScheme and from installScheme.

Reply:
What @fredden meant was:
The current installation code only runs on installation & not on upgrade.
On an extremely rare situation, when someone had a very old version of the yotpo module installed & now upgrading to a newer version, the table won't be created.
So it's not a bad idea to put that on the upgrade script as well, just i case.
Also, I think we should migrate to declarable scheme anyway soon, so it won't matter.

Reply:
Thanks for explaining. I've removed the duplicate code.